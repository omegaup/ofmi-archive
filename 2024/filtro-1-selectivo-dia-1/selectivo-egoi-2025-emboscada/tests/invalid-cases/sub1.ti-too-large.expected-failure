Sub1: 1 <= Ti <= 100
