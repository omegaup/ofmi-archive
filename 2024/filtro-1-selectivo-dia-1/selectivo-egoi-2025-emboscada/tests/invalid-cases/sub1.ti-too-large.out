5 10
1 2 101 4 5
